Cakes-Co/index.html

```diff
@@ -7,10 +7,145 @@
+                <h3>fruit cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.
+                    Vivamus
+                    sit amet felis sodales augue condimentum cursus. Quisque sem erat, congue vel dui a, aliquam porta
+                    est. Phasellus
+                    quis
+                    orci turpis. Morbi pellentesque nisl vitae justo mattis sodales. Suspendisse sagittis et leo eget
+                    rhoncus.</p>
+            </section>
+            <section>
+                <img src="./images/pistachio layer cake.jpg" alt="chocolate-cake" height="500px" width="500px">
+                <h3>Pistachio layer cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.
+                    Vivamus
+                    sit amet felis sodales augue condimentum cursus. Quisque sem erat, congue vel dui a, aliquam porta
+                    est. Phasellus
+                    quis
+                    orci turpis. Morbi pellentesque nisl vitae justo mattis sodales. Suspendisse sagittis et leo eget
+                    rhoncus.</p>
+            </section>
+            <section>
+                <img src="./images/Pound-Cake_.jpg" alt="chocolate-cake" height="500px" width="500px">
+                <h3>Pound cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.
+                    Vivamus
+                    sit amet felis sodales augue condimentum cursus. Quisque sem erat, congue vel dui a, aliquam porta
+                    est.
+                    Phasellus
+                    quis
+                    orci turpis. Morbi pellentesque nisl vitae justo mattis sodales. Suspendisse sagittis et leo eget
+                    rhoncus.</p>
+            </section>
+        </section>
+        <section class="news">
+            <h2>Latest News</h2>
+            <section>
+                <img src="./images/diary-free.jpg" alt="diary-free" width="500px" height="500px">
+                <h3>Pound cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.</p>
+                <label for="read-more-toggle" class="read-more-label">Read More</label>
+            </section>
+            <section>
+                <img src="./images/sugar_free_carrot_cake.jpg" alt="sugar_free_carrot_cake" width="500px"
+                    height="500px">
+                <h3>Pound cake</h3>
+                <p>Duis sit amet rhoncus dui, sit amet feugiat magna. Curabitur eu ligula in orci eleifend dictum nec
+                    quis augue.</p>
+                <label for="read-more-toggle" class="read-more-label">Read More</label>
```

Comment:
this is the label for an element called "read-more-toggle" but I can't find that element.